### PR TITLE
feat: Add FAQ page

### LINKED ISSUE #153: [FEATURE] Add FAQ page
### Detailed description

Hey folks,

I'm proposing to add the FAQ page with 10 prepared questions and answer using Lorem ipsum 😄 Just don't link the FAQ in the navbar for now 

image

We are using TailwindUI and we have a subscription to it, so don't be bothered to use it on our site.

### Context

This will benefit us if there's already a template in our codebase, then we can be able to change the info. once the Q&A is ready

### Possible implementation

You can see the codes here: 
https://tailwindui.com/components/marketing/sections/faq-sections#component-8017f4faee579f7ca518cdde140d4689

### Additional information

Thank you ❤️ 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by AbhinavMV**
#### Description of the change

Closes #153

#### Screenshots

Desktop View
image
image

Mobile View
image

#### Checklist

**I agree to the following :-**

*   Added description of the change
*   I've read the [contributing guidelines](https://github.com/web3community/devprotocol.xyz/blob/main/CONTRIBUTING.md)
*   Search previous suggestions before making a new PR, as yours may be a duplicate.
*   I acknowledge that all my contributions will be made under the project's license.

Notes (**optional**): 

<a href="https://gitpod.io/#https://github.com/web3community/devprotocol.xyz/pull/159"></a>

### CODE REVIEW

File: src/pages/faq.astro
Comment (Panquesito7) [inline src/pages/faq.astro:9]: Missing an endline.
```suggestion
</Layout>

```

Comment (AbhinavMV) [reply src/pages/faq.astro:9]: Will add that.
Also could you tell me why this endline is important?

Comment (Panquesito7) [reply src/pages/faq.astro]: Some IDEs have configured to have that in all files, and not adding that could break those IDEs. 🙂

Comment (AbhinavMV) [reply src/pages/faq.astro]: oohh ok Thanks will keep this in mind.
Also can't we automate this? That is on merging PR all files get an endline if it is not already present? 🤔

**Review by vinzvinci — CHANGES_REQUESTED** [review]
Please add below the Frequently Asked Questions.

If you don't find the answer to your questions, please contact us at [discord](https://discord.gg/VwJp4KM).

File: src/pages/faq.astro
Comment (shuklaritvik06) [reply src/pages/faq.astro]: I think we have editorconfig file and it take care of all these things, you have to install the editorconfig plugin acc to IDE, or code editor you use 🤔

**Comment by shuklaritvik06** [comment]
Please also add this page href in Navbar.vue file

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Maybe add some more padding to the container. It is like aligned to the footer

**Comment by AbhinavMV** [comment]
Added the questions and answers shared.
Added the link to FAQ page in navbar
Added padding between container and footer.

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:104]:
```suggestion
          ans: "Creators can stop using Dev Protocol if they de-authenticate their markets. Stakers can quit if they stop their staking. Unlike memberships of fitness clubs or subscription service, you don't have any particular procedures to quit such as submitting documents.",
```

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:110]:
```suggestion
          ans: "You can stop accepting staking by de-authentication.",
```

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:116]:
```suggestion
          ans: "You can withdraw your Creator Reward at any time. Just pay attention to gas fees, they may vary and can affect your decision to withdraw. There is a yearly limit in place for larger creators which you can learn more about by looking at DIP-55.",
```

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:122]:
```suggestion
          ans: "Project team members can withdraw Creator Reward at any time they want. Just pay attention to gas fees, they may vary and can affect your decision to withdraw. Also, remember that there is a yearly limit in place (DIP-55).",
```

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:134]:
```suggestion
          ans: "Projects that are being developed can be registered. However, development must be recent and ongoing.",
```

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:146]:
```suggestion
          ans: "Only through cryptocurrencies is a complex, unique and sustainable system like Dev Protocol currently possible. The token emission and all the incentives we create are only possible using our token.",
```

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:152]:
```suggestion
          ans: "It is straightforward. Just visit https://stakes.social, connect your wallet find the project you like the most, and stake DEV tokens to sustainably fund them.",
```

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:158]:
```suggestion
          ans: "The best way is by sharing DEV within your community. Explain to them how Dev Protocol allows sustainable funding, how patrons also receive rewards, and how it's different from a simple donation. The bigger the project gets, the more you have the chance for others outside your community to find your project.",
```

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:164]:
```suggestion
          ans: "The gas fees depend on the usage of the Ethereum network. Every operation that happens on this network requires gas to be executed, so the bigger the demand for the network, the bigger the gas fees fill be.",
```

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:176]:
```suggestion
          ans: "To rename or rename a PropertyToken, you can look for the address of the Property on Etherscan and execute changeName / changeSymbol from the Write Contract of the Contract.",
```

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:182]:
```suggestion
          ans: "We don't believe DEV tokens are considered securities for multiple reasons. First, even if creators don't hold any DEV, they can receive DEV as their reward in compensation for their activities. While there is a staking reward built in to incentivize interaction. Users that stake on creators are doing so to signal their importance rather than an expectation of a reward. The DEV token in our eyes is strictly a utility token used to aid the ecosystem.",
```

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:224]:
```suggestion
          ans: "The mission of Dev Protocol is to maximize creativity for every creator of the world, using the power of Blockchain and DeFi to redirect individuals self-interest to sustainably fund projects and creators.",
```

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:230]:
```suggestion
          ans: "You can interact with the smart contract easily with a web experience, it can be anything, from taking a loan to buying an NFT, to funding a creator, the possibilities are endless.",
```

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:236]:
```suggestion
          ans: "To establish a new market, get in touch with the Dev Protocol team, preferably through Discord. After discussions, if the market is approved it will go through the DAO voting to be implemented. ",
```

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:242]:
```suggestion
          ans: "You can withdraw your Staking Reward at any time you want. Just pay attention to gas fees, they may vary and can affect your decision to withdraw. ",
```

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:266]:
```suggestion
          ans: "The DEV Token is the utility token that allows the Dev Protocol ecosystem to exist, through it Creators and Patrons are rewarded.",
```

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:272]:
```suggestion
          ans: "For Patrons, using stakes social allows you to receive rewards while funding your favorite Creator; For Creators, using stakes social allows you to monetize your OSS Project and receive rewards from your Patrons.",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Many spelling errors here. 👀

**Comment by vinzvinci** [comment]
> Please add below the Frequently Asked Questions.
> 
> If you don't find the answer to your questions, please contact us at [discord](https://discord.gg/VwJp4KM).

Don't forget to add thiss

**Review by vinzvinci — CHANGES_REQUESTED** [review]
image
~~Transalate~~ = Translate

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Great catch, @vinzvinci! 👀 Please fix that spelling error mentioned above.

**Comment by AbhinavMV** [comment]
Thanks for catching all the typos. 
@vinzvinci there are many typos in the document as well. Maybe someone can double-check the Japanese Q&A as well.
Also, 2 Japanese Q&A are not complete in the document. One of them has the question but no answer and vice versa for the other one.

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:7]:
```suggestion
      If you don't find the answer to your questions, please contact any of our moderators via
      <a href="https://discord.gg/VwJp4KM" class="underline hover:text-blue-600"
        >Discord</a
```

File: src/components/Faq/Faq.vue
Comment (vinzvinci) [inline src/components/Faq/Faq.vue:3]: can u use the same font here?

like this font 
image

and tell me if you're done adding all the FAQ so we can merge

File: src/components/Faq/Faq.vue
Comment (vinzvinci) [reply src/components/Faq/Faq.vue:7]: Hmm, I think everyone can answer, no need to be specific here, no need to contact mods as they will not report someone.

Comment (Panquesito7) [inline src/components/Faq/Faq.vue:7]: Still needs to be capitalized. 🙂
```suggestion
        >Discord</a
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by vinzvinci** [comment]
> Thanks for catching all the typos. @vinzvinci there are many typos in the document as well. Maybe someone can double-check the Japanese Q&A as well. Also, 2 Japanese Q&A are not complete in the document. One of them has the question but no answer and vice versa for the other one.

I added 2 translations (answer) and 2 questions for Japanese. And some are fixed. Can u check again?

File: src/components/Faq/Faq.vue
Comment (AbhinavMV) [reply src/components/Faq/Faq.vue]: Done 👍

File: src/components/Faq/Faq.vue
Comment (AbhinavMV) [reply src/components/Faq/Faq.vue]: Done 👍

**Comment by AbhinavMV** [comment]
> > Thanks for catching all the typos. @vinzvinci there are many typos in the document as well. Maybe someone can double-check the Japanese Q&A as well. Also, 2 Japanese Q&A are not complete in the document. One of them has the question but no answer and vice versa for the other one.
> 
> I added 2 translations (answer) and 2 questions for Japanese. And some are fixed. Can u check again?

If it is possible could you highlight the cell with the latest changes in blue or any color?
I don't know Japanese so it is tough for me to find changes.

**Comment by vinzvinci** [comment]
> > > Thanks for catching all the typos. @vinzvinci there are many typos in the document as well. Maybe someone can double-check the Japanese Q&A as well. Also, 2 Japanese Q&A are not complete in the document. One of them has the question but no answer and vice versa for the other one.
> > 
> > 
> > I added 2 translations (answer) and 2 questions for Japanese. And some are fixed. Can u check again?
> 
> If it is possible could you highlight the cell with the latest changes in blue or any color? I don't know Japanese so it is tough for me to find changes.

Added. Thanks for the DM.

**Comment by AbhinavMV** [comment]
Thanks for letting me know the missed ques. I have added it as well.
All questions are now added 👍

**Review by vinzvinci — DISMISSED** [review]
LGTM 👍 let's wait for 1 approval and I'll merge this.

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:278]:
```suggestion
          ques: "What are the benefits of using stakes.social?",
```

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:273]:
```suggestion
          ans: "The DEV Token is the utility token that allows the Dev Protocol ecosystem to exist, though, its Creators and Patrons are rewarded.",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: src/components/Faq/Faq.vue
Comment (vinzvinci) [reply src/components/Faq/Faq.vue:273]: Wrong. 

correct:
**through it,**

Comment (Panquesito7) [reply src/components/Faq/Faq.vue:273]: Updated. 🙂

File: src/components/Faq/Faq.vue
Comment (Panquesito7) [inline src/components/Faq/Faq.vue:99]:
```suggestion
          ans: "The Dev Protocol is a way for creators who run projects such as OSS and Open Access to earn income opportunities and achieve sustainability. The Dev Protocol aims to create a world where innovators can take on challenges and solve problems that might not be financially viable but offer value. Creators who create technically socially valuable projects that might not be economically viable are now able to find an alternative way for funding. Reference article: https://initto.devprotocol.xyz/en/what-is-protocol/",
```

**Review by shuklaritvik06 — DISMISSED** [review]
LGTM

**Comment by vinzvinci** [comment]
This is making longer, please try to review one at a time.

**Comment by shuklaritvik06** [comment]
For me the PR looks good acc to the design you discussed and the typos are fixed now I think 🤔

**Comment by shuklaritvik06** [comment]
@vinzvinci  I discussed about this PR it looks good to me and @Panquesito7  , there is one typo, which david suggested in the above comment , you can commit the change and merge if looks good to you

File: src/components/Faq/Faq.vue
Comment (vinzvinci) [reply src/components/Faq/Faq.vue]: fixed

**Comment by vinzvinci** [comment]
> @vinzvinci I discussed about this PR it looks good to me and @Panquesito7 , there is one typo, which david suggested in the above comment , you can commit the change and merge if looks good to you

Thanks, fixed now.

**Review by shuklaritvik06 — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by vinzvinci — APPROVED** [review]
Time to merge 🎉 

If there's any more problem, you can raise an issue [here](https://github.com/web3community/devprotocol.xyz/issues). :octocat:

**Comment by vinzvinci** [comment]
> Maybe add some more padding to the container. It is like aligned to the footer

Hmm, I think this is fixed now.

**Review by krishguptadev — COMMENTED** [review]
Looks good to me 🚀

**Pull request merged**
